examples/reference/menus/SplitButton.ipynb

```diff
@@ -84,15 +84,15 @@
    "outputs": [],
    "source": [
     "split_button = pmui.SplitButton(items=[\n",
-    "    {'label': 'Download .parquet'},\n",
-    "    {'label': 'Download .xlsx'},\n",
-    "], label='Download .csv')\n",
+    "    {'label': 'Save'},\n",
```

Comment:
`SplitButton.loading=True` not working

<img width="200" height="89" alt="image" src="https://github.com/user-attachments/assets/b8c126f0-3c5f-479f-8fc5-3f623316454f" />

```python
import panel as pn
import panel_material_ui as pmui

pn.extension()

pmui.SplitButton(label='Loading', items=[{'label': 'A'}], loading=True).servable()
```